go.mod

```diff
@@ -58,6 +58,7 @@ require (
 	k8s.io/metrics v0.25.6
 	k8s.io/utils v0.0.0-20220728103510-ee6ede2d64ed
 	sigs.k8s.io/controller-runtime v0.12.2
+	sigs.k8s.io/json v0.0.0-20220713155537-f223a00ba0e2
```

Comment:
Can we update this too?
https://pkg.go.dev/sigs.k8s.io/json?tab=versions

Latest seems to be `
v0.0.0-20221116044647-bc3834ca7abd`

Reply:
done